<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="checkBox_toggleZoom">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Toogle Zooming region and second sub-plot</string>
     </property>
     <property name="statusTip">
      <string>Toogle Zooming region and second sub-plot</string>
     </property>
     <property name="text">
      <string>Zoom region</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_toggleCrosshair">
     <property name="text">
      <string>Crosshair</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_toggleLegend">
     <property name="toolTip">
      <string>Toggle plot legend</string>
     </property>
     <property name="statusTip">
      <string>Toggle plot legend</string>
     </property>
     <property name="text">
      <string>Legend</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_togglePoints">
     <property name="toolTip">
      <string>Toggle visualization of data-points</string>
     </property>
     <property name="statusTip">
      <string>Toggle visualization of data-points</string>
     </property>
     <property name="text">
      <string>Points</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_plot">
     <property name="toolTip">
      <string>Toggle window with plot items</string>
     </property>
     <property name="statusTip">
      <string>Toggle window with plot items</string>
     </property>
     <property name="text">
      <string>Plot</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
